packages/block-editor/src/components/block-list/style.scss

```diff
@@ -198,6 +199,18 @@
 		min-height: ( $block-padding + $block-spacing ) * 2;
 	}
 
+	&::after {
+		content: "";
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		border-radius: $radius-block-ui;
+		transition: box-shadow 0.1s ease-in;
```

Comment:
Can you include `@include reduce-motion("transition");` after this one? That should be sufficient to disable it for people who choose to.

Reply:
Yes, good call. There's also probably some redundancy on the initial styles declaration now that the default ::after includes it.